security/rbac/docs/index.rst

```diff
@@ -146,11 +146,11 @@ Broker
    .. code:: bash
    
       # Broker Admin
-      confluent-v1 iam rolebinding create --principal User:$USER_ADMIN_SYSTEM --role SystemAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID
+      confluent iam rbac create --principal User:$USER_ADMIN_SYSTEM --role SystemAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID
```

Comment:
https://docs.confluent.io/confluent-cli/current/command-reference/iam/rbac/role-binding/confluent_iam_rbac_role-binding_create.html#confluent-iam-rbac-role-binding-create
```suggestion
      confluent iam rbac role-binding create --principal User:$USER_ADMIN_SYSTEM --role SystemAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID
```

Reply:
Ack and done in latest commit